### ISSUE TITLE
Recover for real image

### ISSUE THREAD

**Issue by Harxis** [ldz666666/RiDDLE#4]
Thank you for your excellent work. I noticed that you actually use the w-code generated images to do the comparison of the recovered effect during the training and testing, why don't you use the recovered image to do the comparison with the original real image?

**Comment by DongzeLi-CASIA** [comment]
During training, we calculate the loss between the inverted image (w code generated) and the generated ones, because they are at the same domain. 
You can paste  the generated face back to the original background using a pretrained face parsing network in [celebamask-hq](https://github.com/switchablenorms/CelebAMask-HQ).
